WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift

```diff
@@ -156,20 +156,34 @@ import WordPressKit
     ///   - onLoginButtonTapped: Called when the login button on the prologue screen is tapped.
     ///   - onCompletion: Called when the login UI presentation completes.
     public class func showLogin(from presenter: UIViewController, animated: Bool, showCancel: Bool = false, restrictToWPCom: Bool = false, onLoginButtonTapped: (() -> Void)? = nil, onCompletion: (() -> Void)? = nil) {
-        defer {
-            trackOpenedLogin()
+        guard let loginViewController = loginUI(showCancel: showCancel, restrictToWPCom: restrictToWPCom, onLoginButtonTapped: onLoginButtonTapped) else {
```

Comment:
So if I understand correctly, this is the method that WordPress would call to initialize the login flow, while WooCommerce would call `loginUI`?

If that's the case, would it make sense to make the names of both methods a bit more explicit, or rename in a way that the method name indicates that one method presents the login flow while the other leaves the responsibility of presenting the flow to the client app?

I'm curious about what others think though

Reply:
@ctarda - My personal opinion is that we'll probably want to follow suit in WPiOS soon (in showing the login UI as the root VC).  There are several advantages in showing the login UI as the root VC.

With this in mind I'd say the naming isn't overly important as we'll eventually deprecate `showLogin(...)` entirely - the documentation of these methods should be enough for starters.

Reply:
Correction: we may never deprecate `showLogin(...)` as it is currently used in WPiOS to log in to additional sites (which means it wouldn't be shown as the root VC).

Still the naming `showLogin(...)` and `loginUI(...)` sounds good to me.  The other only option I can think of is to use `loginViewController(...)` instead of `loginUI(...)` but I'm not sure that'd make a huge difference.

Reply:
Sounds good!